### ISSUE TITLE
Fix initial skill choice

### ISSUE THREAD

**Issue by johnkslg** [StrangeLoopGames/EcoIssues#19614]
Something we've talked about a lot, you can choose skills that end up not being useful. 

Feedback from player:
"For feedback, it was easy for me to lock myself into mismatched skills.  This is particularly true on a public server with cheap scrolls.  It feels to me a little bit like the issue in Settlers of Catan - where my first choices are some of the most important ones I’ll make, but I have to choose before I know good choices look like."

(if this is a dupe, add above to it and prioritize it)

**Comment by SlayksWood** [comment]
I don't think we need to do it with the current skill system. This will have a bad effect on the balance.
